### PR TITLE
Adding 1 Website

### PR DESCRIPTION
**Pull request by C3n7ral051nt4g3ncy**
## Added `1 site`:

- `Chaturbate`
- Category: `Porn`

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -998,6 +998,18 @@
...
+        "valid" : true
+       },        
+       {
+        "name" : "chaturbate",
+        "uri_check" : "https://chaturbate.com/{account}/",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "'s Bio and Free Webcam",
+        "m_string" : "It's probably just a broken link",
+        "m_code" : 302,
```
Comment (WebBreacher) [inline wmn-data.json:1010]: Can you check the `m_code` as I'm only seeing 404 and not 302?

Comment (C3n7ral051nt4g3ncy) [reply wmn-data.json:1010]: @WebBreacher First I got a `404`, tested that with the python script and there was false positives every try, then I put the link through this tool: https://www.webfx.com/tools/http-status-tool 
this tool returned a 302, I then changed the m_code to 302, this gave a good detection via the python script.
But no worries, I can change it to 404 if it's good your end.

Comment (C3n7ral051nt4g3ncy) [reply wmn-data.json]: @WebBreacher  `404 Change`:  Done ✅

**Comment by WebBreacher** [comment]
Unfort. there is a conflict in the minified JSON file. I've reworked the GitHub Action to only create this file AFTER merge instead of at PR. Can you resubmit this PR?

**Pull request closed without merging**